### ISSUE TITLE
Memory-leaks

### ISSUE THREAD

**Issue by aminpaks** [angular-redux/example-app#34]
I changed two lines in your example and run it. After about 30,000 actions app accumulates up to 300MB memory and crashes.

These are steps to reproduce this issue:
1. Update `addTicket` in `animal` component to, this code simulates dispatching many actions to observe the issue easier:
```
  addTicket = () => {
    for (let i = 0; i < 10000; i++) {
      setTimeout(() => this.subStore.dispatch({ type: 'ADD_TICKET' }), 1);
    }
  }
```

2. Remove the `createLogger` and `devTools enhancer` from `store/module`:
```
    // Tell Redux about our reducers and epics. If the Redux DevTools
    // chrome extension is available in the browser, tell Redux about
    // it too.
    store.configureStore(
      rootReducer,
      {},
      [ ...rootEpics.createEpics() ]);
```

3. Run the app `yarn start`
4. Click on `+` to start the process: this will dispatch 10000 actions and update the UI.
5. Observe how app crashes after a few times clicking on the plus

You may check the memory in `Chrome devTools` too.

**Comment by aminpaks** [comment]
No comment at all, anyone!?

**Comment by aminpaks** [comment]
I already fixed the issue in `angular-redux/store` project, where can I ask for PR?

**Comment by SethDavenport** [comment]
Hey - looking into it now. Please understand that I don't get to everything immediately.

**Comment by SethDavenport** [comment]
I've reproduced the issue and confirmed that your fix works. Thanks, nice catch!

I also took the liberty of creating a PR with from your fork, here: https://github.com/angular-redux/store/pull/409.  There are a couple of minor cleanup items after which I'd be happy to merge it.

**Comment by SethDavenport** [comment]
I cleaned up the PR and merged it. The fix has been released in `@angular-redux/store` 6.4.1.
